stdlib/tests/crypto/falcon.rs

```diff
@@ -72,10 +72,9 @@ pub fn push_falcon_signature(process: &mut ProcessState) -> Result<(), EventErro
     let result = falcon_sign(pk_sk, msg)
         .ok_or(FalconError::MalformedSignatureKey { key_type: "RPO Falcon512" })?;
 
-    for r in result {
-        process.advice_provider_mut().push_stack(r);
-    }
-    Ok(())
+    Ok(Vec::from_iter(
+        result.into_iter().map(|value| AdviceMutation::PushStack { value }),
+    ))
```

Comment:
We should be able to do this as a single mutation: `AdviceMutation::ExtendStack(result)`.